File: Resueltos_por_competidor/Juan_Martinez/YoungPhysicist.cpp
/*
 * Autor: Juan Martinez
 * Problema: Young Physicist (69A)
 * Juez online: Codeforces
 * Veredicto: Accepted
 * Url: https://codeforces.com/problemset/problem/69/A
 * Difficulty: 1000
 */
#include <bits/stdc++.h>

using namespace std;

int main()
{
    int n, res[6];
    scanf("%d", &n);
    res[3] = 0;
    res[4] = 0;
    res[5] = 0;
    for(int i = 0; i < n; i++) {
            scanf("%d %d %d", &res[0], &res[1], &res[2]);
            res[3] += res[0];
            res[4] += res[1];
            res[5] += res[2];
    }
    if(res[3] == 0 && res[4] == 0 && res[5] == 0) printf("%s", "YES");
    else printf("%s", "NO");
    return 0;
}

File: Resueltos_por_competidor/Santiago_Ortiz/YoungPhysicist.java
import java.util.Scanner;

/*
 * Autor: Darkovski7   
 * Problema: Young Physicist
 * Juez Online: Codeforces
 * Veredicto: Accepted 
 * Url: https://codeforces.com/problemset/problem/69/A
 */

public class YoungPhysicist {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);

		int n = sc.nextInt();
		int sumaX = 0;
		int sumaY = 0;
		int sumaZ = 0;

		for (int i = 0; i < n; i++) {
			sumaX += sc.nextInt();
			sumaY += sc.nextInt();
			sumaZ += sc.nextInt();

		}

		if (sumaX == 0 && sumaY == 0 && sumaZ == 0) {
			System.out.println("YES");
		} else {
			System.out.println("NO");
		}
	}

}
